### ISSUE TITLE
Point map icons 

### ISSUE THREAD

**Issue by trosendal** [SVA-SE/svamap#49]
I think we need to come up with at solution to show positive samples with a different geometric shape. Only colour does not do the trick for colorblind people.

What if the positives should be upside down red triangles? That increases in size with many positive samples in the same geographical spot?

**Comment by trosendal** [comment]
We can investigate this but I think it might be difficult to adjust the size relative to the number of cases. We are using the radius of an L.circlemarker to define the size of the marker:

https://github.com/SVA-SE/svamap/blob/master/inst/influenza/map.html#L109

where using an icon like a triangle.png on the map also can also be scaled:

http://leafletjs.com/reference.html#icon

But the scale of the 'radius' in the first and 'iconSize' in the second may be different. Not sure if they both are in pixels? 

I think if we want to implement different shapes it would be good to go with only 'L.icon' for both circles and triangles. Then the scaling will match.

**Comment by trosendal** [comment]
Downside to using the L.icon approach is that the color cannot be dynamically defined.

**Comment by trosendal** [comment]
This subject was brought up again today internally. Because we need to display two properties on the maps (Number of cases with the size and state of the cases with colour) then using different shapes is not possible. But perhaps just changing the colour scheme to use colour blind safe colours and it's solved. There is a tool available for selecting such colours:

http://colorbrewer2.org/#type=diverging&scheme=PuOr&n=3

**Comment by trosendal** [comment]
Discussion of marker sizes of different shapes:

http://graphicdesign.stackexchange.com/questions/36730/how-to-make-plot-marker-sizes-for-scientific-plots-seem-consistent

**Comment by Gcocca** [comment]
This approach sound interesting (Line 24-)
[https://github.com/rstudio/leaflet/blob/master/inst/examples/icons.R#L24](url)

**Comment by trosendal** [comment]
Thanks, that is a good idea. Then the icon colour could be dynamically defined in the R script. I wonder about using an alpha value in the colour to allow for semitransparency and therefore it could work for overlapping icons.

**Comment by trosendal** [comment]
I have changed the colours and alpha values of the icons for the two deployed maps to use colours derived from colourbrewer that are apparently colourblind safe and highly contrasting. I have also added a black edge to the icons in order to improve contrast with the background. Unfortunately, when I look at the before and after I liked the before better. I think this may become a competition between pure aesthetics and readability. See https://github.com/SVA-SE/svamap/commit/cc1253985eab925e77a8b5558a15d40101ebd7aa to study the before and after colour choices
